tests/drivers/spi/spi_error_cases/boards/nrf54h20dk_nrf54h20_cpuapp.overlay

```diff
@@ -0,0 +1,83 @@
+/*
+ * Copyright (c) 2024 Nordic Semiconductor
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/delete-node/ &exmif;
```

Comment:
Maybe it is worth to mention that this line is here only due to https://github.com/zephyrproject-rtos/zephyr/issues/72657

Reply:
I added a comment